presto-bytecode/src/main/java/com/facebook/presto/bytecode/MethodDefinition.java

```diff
@@ -70,6 +72,8 @@ public MethodDefinition(
             Iterable<Parameter> parameters
     )
     {
+        checkArgument(Iterables.size(parameters) <= 254, "Too many parameters for method");
```

Comment:
Since it MethodDefinition, it implies this pointer will consume one available parameter.